### PR TITLE
scss compile, expanded for dev

### PR DESCRIPTION
**Pull request by sean-au**

### CODE REVIEW

**Comment by sean-au** [comment]
ouptuts explanded SCSS in dev env
dev env now includes development and local
scss error message supressed in production (only basic message shown)

File: inc/scss-compiler.php [hunk 1]
```diff
@@ -62,7 +63,11 @@ function bootscore_compile_scss() {
       set_theme_mod('bootscore_scss_modified_timestamp', $last_modified);
     }
   } catch (Exception $e) {
-    wp_die('<b>bootScore SCSS Compiler - Caught exception:</b><br><br> ' . $e->getMessage());
+    if ($is_environment_dev) {
+      wp_die('<b>bootScore SCSS Compiler - Caught exception:</b><br><br> ' . $e->getMessage());
+    } else { // supress error message for production
+      wp_die('<b>SCSS ERROR!</b>');
```
Comment (justinkruit) [inline inc/scss-compiler.php:69]: A nicer error message would be better.
```suggestion
      wp_die('Something went wrong with the SCSS compiler.');
```

File: inc/scss-compiler.php [hunk 2]
```diff
@@ -62,7 +63,11 @@ function bootscore_compile_scss() {
       set_theme_mod('bootscore_scss_modified_timestamp', $last_modified);
     }
   } catch (Exception $e) {
-    wp_die('<b>bootScore SCSS Compiler - Caught exception:</b><br><br> ' . $e->getMessage());
+    if ($is_environment_dev) {
+      wp_die('<b>bootScore SCSS Compiler - Caught exception:</b><br><br> ' . $e->getMessage());
+    } else { // supress error message for production
```
Comment (justinkruit) [inline inc/scss-compiler.php:68]: The comment isn't really necessary.
```suggestion
    } else {
```

**Review by justinkruit — CHANGES_REQUESTED** [review]
If you could change these things, then I think it's fine for approval.

**Comment by sean-au** [comment]
Hi @justinkruit I pushed the requested changes

**Review by justinkruit — APPROVED** [review]
All looks good now 👍

**Pull request merged**